CMakeLists.txt

```diff
@@ -1496,9 +1497,20 @@ ldconfig
 rm -f -- /etc/ld.so.conf.d/libfluent-bit.conf
 ldconfig
     ")
-    set(CPACK_DEBIAN_RUNTIME_PACKAGE_CONTROL_EXTRA "${PROJECT_BINARY_DIR}/scripts/postinst;${PROJECT_BINARY_DIR}/scripts/prerm")
+    list(APPEND CPACK_DEBIAN_RUNTIME_PACKAGE_CONTROL_EXTRA "${PROJECT_BINARY_DIR}/scripts/postinst;${PROJECT_BINARY_DIR}/scripts/prerm")
   endif(FLB_RUN_LDCONFIG)
 
+  file(WRITE ${PROJECT_BINARY_DIR}/scripts/postinst "#!/bin/sh
```

Comment:
Rather than writing files I prefer to have them detained to make it easier to review/figure out what is going on rather than delving into cmake. Is that possible here?

If not, would single quotes mean we can remove the escaping later which is always problematic and easy to mess up or forget?